views/index.ejs

```diff
@@ -0,0 +1,7 @@
+<h1>Link Shortener</h1>
+<p>Copy and paste a link in the text field to generate a shorer one.</p>
+<form method="POST" action="/links">
+  <label for="url">URL</label>
+  <input type="text" name="url" placeholder="enter URL here">
```

Comment:
One issue is that if the user doesn't add the http:// to their link the redirect command doesn't work and you get an error with the link. You can do a front end check for a vlaid http:// web address by setting the type to url instead of text type="url